src/Adapters/DatabasePendingApprovalRetriever.php

```diff
@@ -70,6 +70,7 @@ private function queryPendingApprovals( array $categories ): IResultWrapper {
 			__METHOD__,
 			[
 				'GROUP BY' => 'page_id',
+				'ORDER BY' => 'rev_timestamp DESC',
```

Comment:
I ended up not testing this, since our current test approach won't work for it (stuff is created at the same time), and creating something else does not seem worth it